### DISCUSSION TITLE
unusable

### DISCUSSION THREAD

**Discussion by gregcault** [ical4j/ical4j#521] [Ideas]
I don't want to spend hours trying to figure out where to put a ical4.properties, when the only reason I need a ical4.properties is to prevent the calendar breaking due to a dependency on an 3rd party cache library that is something I will never install.  Maybe remove dependency and the next person can use the software out the box.

**Comment by benfortuna** [comment]
FYI the next release (3.1.2) will default to using the Map cache implementation.
